migrations/sqlite/20250415100644_device_codes.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE device_codes
+(
+    device_code            VARCHAR(36) NOT NULL PRIMARY KEY,
+    device_code_created_at TEXT        NOT NULL,
+    device_code_expires_at TEXT        NOT NULL,
+    token_iat              INTEGER,
+    token_exp              INTEGER,
+    token_last_used_at     TEXT,
+    account_id             VARCHAR(100) REFERENCES accounts (id)
+);
+
+CREATE INDEX idx_device_codes_device_code_expires_at
+    ON device_codes (DATETIME(device_code_expires_at) DESC);
```

Comment:
It's interesting. Please double check if we apply this trick in other tables where sorting by time might be expected.

Reply:
`accounts` & `accounts_passwords` tables were updated (SQLite, PostgreSQL). In the case of MySQL, on the contrary, `LOWER()` calls have been removed, since columns are already case-insensitive by default.

Done